dotCMS/src/main/java/com/dotcms/auth/providers/jwt/factories/ApiTokenAPI.java

```diff
@@ -473,4 +478,21 @@ private boolean checkPerms(final ApiToken token, final User user) {
                 || APILocator.getPortletAPI().hasUserAdminRights(user))
                 .getOrElse(false);
     }
+
+    /**
+     * Return true if the token was correctly constructed
+     * @param token
+     * @return
+     */
+    public  boolean isWellFormedToken(final String token){
+        try {
+            Jwts.parser().parse(token);
+            return true;
+        } catch (MalformedJwtException e){
+            return false;
+        } catch(SignatureException | ExpiredJwtException | IllegalArgumentException e){
```

Comment:
Codacy found an issue: ['catch' branch identical to 'MalformedJwtException' branch](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6190239)